### ISSUE TITLE
sentry-cli is not detected in windows

### ISSUE THREAD

**Issue by rchamorro** [getsentry/sentry-fastlane-plugin#70]
We are running fastlane sentry plugin from windows and sentry-cli is not detected even if it is installed.

I think it is because in windows which command does not exist (it is called "where") and that is used to detect if it is installed.

https://github.com/getsentry/sentry-fastlane-plugin/blob/2e0c697d1e0b96704dc574d3dccf4c3ccdf9812e/lib/fastlane/plugin/sentry/helper/sentry_helper.rb#L5

It should try to detect sentry-cli presence using a cross platform compatible method, like trying to run sentry-cli without and check for known texts.

Another possibility is to allow passing the sentry-cli executable path like what is proposed in this PR:

https://github.com/getsentry/sentry-fastlane-plugin/pull/42

**Comment by Tanuja0049** [comment]
Hi Team,

We also want to use this plugin to upload the source maps in sentry. Is there any update on this issue?

**Comment by kohenkatz** [comment]
I think the simplest way to fix this is to replace:

```
unless `which sentry-cli`
```

with:

```
require 'ptools'
if File.which('sentry-cli').nil?
```

The [`ptools` gem](https://rubygems.org/gems/ptools/) knows how to handle this cross-platform.

**Comment by kohenkatz** [comment]
It looks like this should be fixed by #97, which was recently merged.

**Comment by philipphofmann** [comment]
Yes, indeed, @kohenkatz. At least with https://github.com/getsentry/sentry-fastlane-plugin/pull/97 you should have a workaround.
